### PR TITLE
PropertyDDS: fix local tip view update

### PR DESCRIPTION
**Pull request by evaliyev**

### CODE REVIEW

**Comment by evaliyev** [comment]
@ruiterr @nedalhy

**Comment by tylerbutler** [comment]
I assigned this to myself as a reminder to merge once ready.

File: experimental/PropertyDDS/packages/property-dds/src/propertyTree.ts [hunk 1]
```diff
@@ -100,592 +100,594 @@ export interface SharedPropertyTreeOptions {
...
+
+        // Update the tip view
+        if (!this.tipView) {
+            this.tipView = cloneDeep(this.remoteTipView);
+            const changeSet = new ChangeSet(this.tipView);
+            changeSet.applyChangeSet(accumulatedChanges);
+        } else {
+            const viewCS = new ChangeSet(this.tipView);
+            viewCS.applyChangeSet(newTipDelta);
+            viewCS.applyChangeSet(pendingChanges);
```
Comment (nedalhy) [inline experimental/PropertyDDS/packages/property-dds/src/propertyTree.ts:684]: Can you please add a reproducible test ?

**Comment by dstanesc** [comment]
@evaliyev @ruiterr @nedalhy Wondering whether this defect was addressed elsewhere or still a problem?

**Pull request closed without merging**